package-lock.json

```diff
@@ -1521,6 +1521,16 @@
         "@babel/types": "^7.3.0"
```

Comment:
There's a package-lock.json and a yarn.lock in this PR. Either yarn _or_ npm should be used, not both, or they can get out of sync.